hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OmMetadataGenerator.java

```diff
@@ -326,8 +339,18 @@ private void applyOperation(long counter) throws Exception {
     case CREATE_KEY:
       keyName = getPath(counter);
       getMetrics().timer(operation.name()).time(() -> {
-        try (OutputStream stream = bucket.createStreamKey(keyName,
-            dataSize.toBytes(), replicationConfig, new HashMap<>())) {
+        try (OutputStream stream = bucket.createKey(keyName,
+            dataSize.toBytes())) {
+          contentGenerator.write(stream);
+        }
+        return null;
+      });
+      break;
+    case CREATE_STREAM_KEY:
+      keyName = getPath(counter);
+      getMetrics().timer(operation.name()).time(() -> {
+        try (OzoneDataStreamOutput stream = bucket.createStreamKey(
+            keyName, dataSize.toBytes(), defaultReplicationConfig, Collections.emptyMap())) {
```

Comment:
already replicationConfig is there, any reason for using defaultReplicationConfig always for this case?

Reply:
Mainly for explicit control of parameters.

Reply:
`replicationConfig` this is based "ozone.replication" with default 3 ratis. Now we are using always fixed param value. Do this needs to be configurable?